db/migrate/20230203073707_create_comment.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComment < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.integer :comments_counter, default: 0
+      t.references :author, null: false, foreign_key: { to_table: :users }
+      t.references :post, null: false, foreign_key: { to_table: :posts }
```

Comment:
- Please change to: `t.references :post, foreign_key: true` as suggested by the previous reviewer

Reply:
Thanks for the review, I have noted the required changes. 
I am unable to request a new review from my dashboard after effecting the changes because it still displays "Review Pending". 
Please advise on what to do.